downstream/modules/platform/ref-controller-database-settings.adoc

```diff
@@ -15,7 +15,24 @@ To improve the performance of the PostgreSQL server, configure the following _Gr
 
 * `shared_buffers`: determines how much memory is dedicated to the server for caching data. The default value for this parameter is 128 MB. When you modify this value, you must set it between 15% and 25% of the machine's total RAM. 
 
-NOTE: You must restart the database server after changing the value for shared_buffers.
+WARNING
+====
+If you are compiling Postgres against OpenSSL 3.2, your system regresses to remove the parameter for User during startup. You can rectify this by using the BIO_get_app_data call instead of open_get_data. Only an administrator can make these changes, but it impacts all users connected to the PostgreSQL database.
+
+If you update your systems without the OpenSSL patch, you are not impacted, and you do not need to take action.
+====
+
+WARNING
```

Comment:
As you mentioned, there's a duplication of the admonition. Can you see it in VSCode? I think we should remove the dup before we merge, if that's possible.

Reply:
Otherwise, LGTM. I can squash/merge when the correction is made.